icecreamshop/src/main/java/com/summerschool/icecreamshop/repository/ProductRepository.java

```diff
@@ -10,4 +10,7 @@
 @Repository
 public interface ProductRepository extends JpaRepository<Product, Long> {
     List<Product> findById(int id);
+
+    @Query(nativeQuery = true, value = "Select * From products Where products.how_many_alergens=0")
+    List<Product> querySpecialProducts();
```

Comment:
E corect. Pe viitor, daca nu vrei sa scrii tu query-ul si extinzi interfata `JpaRepostiory` poti scrie asa:
```java
List<Product> findAllByHowManyAlergens(Integer howManyAlergens);
```
Iar in controller ai putea sa o apelezi `findAllByHowManyAlergens(0)`.